### DISCUSSION TITLE
cellColData defining pseudo-bulk sample for each cell

### DISCUSSION THREAD

**Discussion by WeiWei060512** [GreenleafLab/ArchR#1974] [Questions / Documentation]
Hello, 

I ran addGroupCoverages to create pseudo-bulk samples. Is there a way to get the information which cells are grouped together? e.g a column in cellColData to annotate each cell which pseudo-bulk sample belongs to. Thanks.

**Comment by rcorces** [comment]
https://www.archrproject.com/reference/addGroupCoverages.html#:~:text=for%20parallel%20computing.-,returnGroups,but%20rather%20peaks%20are%20called%20from%20a%20TileMatrix%20(peakMethod%20%3D%20%22Tiles%22).,-parallelParam

**Comment by WeiWei060512** [comment]
Hi Ryan, thanks for your reply. Do you have any suggestions how to get this information? 

Thanks,
Wei
